tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -1251,6 +1362,8 @@ void SpirvEmitter::doVarDecl(const VarDecl *decl) {
     // not have local storage, it should be file scope variable.
     const bool isFileScopeVar = !decl->hasLocalStorage();
 
+    // TODO: if no initializer exists, just emit DebugDeclare for OpVariable.
```

Comment:
Is this still relevant? If yes, please file an issue so we don't forget it.

Reply:
Already handled. I removed the TODO.